src/Workspace/index.jsx

```diff
@@ -266,6 +269,11 @@ class Workspace extends React.Component {
             this.state.notebookStatus !== 'Running' &&
             this.state.notebookStatus !== 'Stopped' ?
               <div>
+                {workspacePageDescription ?
+                  <p className='workspace__description'>
+                    {parse(workspacePageDescription)}
```

Comment:
i'm in favor of give this thing a default description

Reply:
why not but that's not my call to make 🤔